tests/test_crud.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from api import crud
+
+
```

Comment:
You can use 
`pytestmark = pytest.mark.asyncio`
In this file

Reply:
and then remove @pytest.mark.asyncio?

Reply:
Yes

Reply:
will do